security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -35,6 +62,40 @@ Version `1.12.0` of the {cert-manager-operator} is based on the upstream cert-ma
 
 * Previously, the {cert-manager-operator} could not pull the `jetstack-cert-manager-container-v1.12.4-1` image. The cert-manager controller, CA injector, and Webhook pods were stuck in the `ImagePullBackOff` state. Now, the {cert-manager-operator} pulls the `jetstack-cert-manager-container-v1.12.4-1` image to run the cert-manager controller, CA injector, and Webhook pods successfully. (link:https://issues.redhat.com/browse/OCPBUGS-19986[*OCPBUGS-19986*])
 
+[id="cert-manager-operator-release-notes-1.11.5"]
+== Release notes for {cert-manager-operator} 1.11.5
+
+Issued: 2023-10-25
+
+The following advisory is available for the {cert-manager-operator} 1.11.5:
+
+* link:https://access.redhat.com/errata/RHSA-2023:6279-02[RHSA-2023:6279-02]
+
+Version `1.11.5` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.11.5`. For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.11/#v1115[cert-manager project release notes for v1.11.5].
```

Comment:
```suggestion
The golang version is updated to the version `1.20.10` to fix Common Vulnerabilities and Exposures (CVEs). Version `1.11.5` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.11.5`. For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.11/#v1115[cert-manager project release notes for v1.11.5].
```

Reply:
@swghosh I have added the suggested text. Can you provide approval if this looks good?